lib/alerts/util/index.js

```diff
@@ -54,7 +54,10 @@ export const EFFECTS = [
   'ADDITIONAL_SERVICE',
   'MODIFIED_SERVICE',
   'STOP_MOVED',
-  'OTHER_EFFECT'
+  'OTHER_EFFECT',
+  // Items introduced by MTC.
```

Comment:
This comment is incorrect. These are items added in v2 of GTFS-rt: https://gtfs.org/reference/realtime/v2/#enum-effect

Reply:
Just pushed.